src/cli/commands/check.js

```diff
@@ -40,7 +40,7 @@ export async function run(
   }
 
   // get patterns that are installed when running `yarn install`
-  const [, rawPatterns] = await install.hydrate();
+  const [, rawPatterns] = await install.hydrate(true);
```

Comment:
That's a good question. I mean if the package is in the cache, it doesn't actually fetch anything, right?